nipype/pipeline/plugins/base.py

```diff
@@ -197,13 +197,18 @@ def _get_result(self, taskid):
     def _submit_job(self, node, updatehash=False):
         raise NotImplementedError
 
-    def _report_crash(self, node, result=None):
-        tb = None
+    def _report_crash(self, node, result=None, traceback=None):
```

Comment:
is the change in signature because multiproc is not adding traceback to result? i.e., could this remain the same and `'traceback'` added to `result`?